src/main/java/seedu/ddd/model/event/Event.java

```diff
@@ -0,0 +1,112 @@
+package seedu.ddd.model.event;
+
+import static seedu.ddd.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.ddd.commons.util.ToStringBuilder;
+import seedu.ddd.model.contact.client.Client;
+import seedu.ddd.model.contact.vendor.Vendor;
+
+/**
+ * Represents a Event in the address book.
+ * Guarantees: to be determined.
+ */
+public class Event {
+
+    private final ArrayList<Client> clients;
+    private final ArrayList<Vendor> vendors;
+    private final String eventDescription;
+
+    /**
+     * Constructs a {@code Event}.
+     *
+     * @param clients A list of client.
+     * @param vendors A list of vendors.
+     */
+    public Event(ArrayList<Client> clients, ArrayList<Vendor> vendors, String description) {
+        requireAllNonNull(clients, vendors);
+        // another test to check
+        this.clients = clients;
+        this.vendors = vendors;
```

Comment:
We should copy instead of assigning since the lists will be passed as references.